src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.AnalyzerExecutionContext.cs

```diff
@@ -204,13 +204,10 @@ void processMembers(IEnumerable<ISymbol> members)
                                 memberSet ??= new HashSet<ISymbol>();
                                 memberSet.Add(member);
 
-                                // Ensure that we include symbols for both parts of partial methods.
-                                // https://github.com/dotnet/roslyn/issues/73772: also cascade to partial property implementation part
-                                if (member is IMethodSymbol method &&
-                                    !(method.PartialImplementationPart is null))
-                                {
-                                    memberSet.Add(method.PartialImplementationPart);
-                                }
+                                if (member is IMethodSymbol { PartialImplementationPart: { } methodImplementation })
+                                    memberSet.Add(methodImplementation);
+                                else if (member is IPropertySymbol { PartialImplementationPart: { } propertyImplementation })
```

Comment:
Will we need to fix this up for `field`, with property initializers on the implementation part?

Reply:
I don't think this code would need to do anything differently based on the implementation part having an initializer. We also don't expect any backing field to be included in the set because `IsImplicitlyDeclared` will always be `true` on it.